### PR TITLE
Update test/ user schemas with local copies

### PR DESCRIPTION
**Pull request by kimlisa**
- created a script to run a local postgres container for integration test
- tested script with integration tests in `api-blocks`

### CODE REVIEW

File: db-local-run/run_hwsc_user_svc_db.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+USER=$hosts_postgres_user
+SSLMODE=$hosts_postgres_sslmode
+
+MIGRATION_PATH=../db-migrations/hwsc-user-svc/test/psql/
+POSTGRES_URI=postgres://$USER:$PASSWORD@$HOST:$PORT/$DB?sslmode=$SSLMODE
+
+echo "Launching a local postgres container for hwsc-user-svc:"
+CONTAINER_ID=$(docker run -d -p 5432:5432 --rm -e POSTGRES_PASSWORD=$PASSWORD -e POSTGRES_DB=$DB postgres:alpine)
+echo $CONTAINER_ID
+sleep 3;
```
Comment (faraonc) [inline db-local-run/run_hwsc_user_svc_db.sh:17]: can you write an issue for this?

File: db-local-run/run_hwsc_user_svc_db.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+echo
+echo "Running all psql migration files:"
+migrate -path $MIGRATION_PATH -database $POSTGRES_URI up
```
Comment (faraonc) [inline db-local-run/run_hwsc_user_svc_db.sh:20]: In the future becareful of not placing a number after `up`

**Review by faraonc — APPROVED** [review]
TU

File: db-local-run/run_hwsc_user_svc_db.sh (hunk 1 above)
Comment (kimlisa) [reply db-local-run/run_hwsc_user_svc_db.sh:17]: https://github.com/hwsc-org/hwsc-dev-ops/issues/20

**Pull request merged**